example/mqtt2sesame/mqtt2sesame.py

```diff
@@ -37,16 +53,19 @@ async def connect_sesame(ble_device_identifier: str, **kwargs) -> CHDevices:
         raise ConnectionRefusedError("Failed in scanning the device.")
 
     device.setDeviceStatusCallback(onSesameStateChanged)
-    device.getKey().setSecretKey(kwargs.get("secret_key"))
-    device.getKey().setSesame2PublicKey(kwargs.get("public_key"))
+
+    device.getKey().setSecretKey(s_key)
+    device.getKey().setSesame2PublicKey(p_key)
 
     await device.connect()
     await device.wait_for_login()
 
     return device
 
 
-async def runner_connect_sesame(ble_uuids: list = None) -> Dict[str, CHDevices]:
+async def runner_connect_sesame(
+    ble_uuids: list = None,
```

Comment:
`None` cannot be assigned to type `list[Unknown]`

Reply:
Line 197: `new_connection = await runner_connect_sesame(ble_uuid)` has an issue as well since `ble_uuid` is `str`, however, the type defined here is `list`.

Reply:
Good catch, thank you @zhyu. I changed this variable name to something more understandable and added more typing.

Reply:
Great changes. Thanks for updating the typing!